src/index.html

```diff
@@ -22,6 +22,404 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="moyo_catalog"></div>
+    <header class="header">
+      <a
+        href="#"
+        class="moyo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_item__link z"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card-hover"
+      >
+        <img
+          src="../src/images/imac.jpeg"
+          class="card_imac"
+          alt="iMac"
+        />
+        <p class="card_name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card_code">Product code: 195434</p>
+        <div class="card_reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card_comments">Reviews: 5</p>
+        </div>
+        <div class="card_price">
+          <p class="card_text">Price:</p>
+          <p class="card_dollars">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="buy"
+        >
+          buy
+        </a>
```

Comment:
The href attribute is set to '#', which might not be functional. Consider providing a valid URL or ensuring this is intentional for navigation purposes.